clustered/client/src/main/java/org/ehcache/clustered/client/config/ClusteringServiceConfiguration.java

```diff
@@ -40,6 +41,8 @@
     implements ServiceCreationConfiguration<ClusteringService>,
     CacheManagerConfiguration<PersistentCacheManager> {
 
+  private static final String CLUSTER_URI_PATTERN = "terracotta://([^\\]\\[/?#@]+@)?[^:?#/]+(:[1-9][0-9]{0,4})?(/[^\\?#]*)?(\\?[^#]*)?(#.*)?";
```

Comment:
Something unrelated to this issue exactly: Do we really want to keep this scheme prefix as `terracotta`? Isn't that a bit too long? `tc://` like old times would have been much better I feel.